en/getting_started/px4_basic_concepts.md

```diff
@@ -128,13 +128,16 @@ The companion computer will usually communicate using a MAVLink API like the MAV
 * [Robotics APIs](https://dev.px4.io/master/en/robotics/) (PX4 Developer Guide)
 
 
-## Removable Memory/Logging
+## SD Cards (Removable Memory) {#sd_cards}
 
-PX4 uses SD memory cards for storing [flight logs](../getting_started/flight_reporting.md) (SD support may not be present on every flight controller).
+PX4 uses SD memory cards for storing [flight logs](../getting_started/flight_reporting.md), and they are also required in order to use UAVCAN peripherals and fly [missions](../flying/missions.md).
+
+By default, if no SD card is present PX4 will beep twice during boot (and none of the above features will be available).
```

Comment:
Can you tell me which tune is played? https://docs.px4.io/master/en/getting_started/tunes.html

Reply:
Tune 17, SD_ERROR. Similar (if not the same) to https://docs.px4.io/master/en/getting_started/tunes.html#error-tune.